src/graphics/texture.js

```diff
@@ -65,6 +65,8 @@ import {
  * @param {string} [options.type] - Specifies the image type, see {@link pc.TEXTURETYPE_DEFAULT}
  * @param {boolean} [options.fixCubemapSeams] - Specifies whether this cubemap texture requires special
  * seam fixing shader code to look right. Defaults to false.
+ * @param {boolean} [options.isRenderTarget] - Specifies whether texture or cubemap was used as a render target.
+ * Defaults to false.
```

Comment:
Why does this need to be public API?

Reply:
It doesn't - so I have just made it private